gtests/net/mptcp/common/set_sysctls.py

```diff
@@ -1,21 +1,21 @@
 #!/usr/bin/python
 
-import commands
+import subprocess
 import os
 import sys
 
 pppid = int(os.popen("ps -p %d -oppid=" % os.getppid()).read().strip())
 filename = '/tmp/sysctl_restore_%d.sh' % pppid
 
 restore_file = open(filename, 'w')
-print >> restore_file, '#!/bin/bash'
+restore_file.write('#!/bin/bash')
 
 for a in sys.argv[1:]:
   sysctl = a.split('=')
 
   # save current value
   cur_val = commands.getoutput('cat ' + sysctl[0])
-  print >> restore_file, 'echo "%s" > %s' % (cur_val, sysctl[0])
+  restore_file.write('echo "%s" > %s' % (cur_val, sysctl[0]))
```

Comment:
do you not need an explicit `restore_file.close()` after this loop now? I don't know if it will create the file before the `chmod`

(note: maybe you don't need to use this helper? See below)

Reply:
> do you not need an explicit `restore_file.close()` after this loop now? I don't know if it will create the file before the `chmod`

the file should be created at open() time.